packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -424,14 +447,14 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
                     <Box>
                       <Heading type="title2">{label}</Heading>
                     </Box>
-                    <ModalCloseButton onClick={handleClose} title="Close" />
+                    <ModalCloseButton onClick={_ev => handleClose()} title="Close" />
```

Comment:
unfortunately title here has to be prop, in order to provide translations

Reply:
Right, `labelClose` can be used